### PR TITLE
Beepsky will now save the station by stuncuffing ops.

### PR DESCRIPTION
**Pull request by Tacolizard**
:cl: Beepsky VS Ops
add: Securitrons now hunt down people wearing syndicate hardsuits or masks
tweak: Agent ID cards no longer decrease threat level by 5
/:cl:

after a round of spamming out beepskys because I thought they went after ops, i went into a rage and decided this had to be my first pr

### CODE REVIEW

**Comment by LanCartwright** [comment]
No.

**Comment by Tacolizard** [comment]
wew

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -219,8 +219,8 @@
 				if(!I || !L || I.loc != src || !(I in L.embedded_objects))
 					return
 				L.embedded_objects -= I
-				L.receive_damage(I.embedded_unsafe_removal_pain_multiplier*I.w_class)//It hurts to rip it out, get surgery you dingus.
-				I.forceMove(get_turf(src))
+				L.take_damage(I.embedded_unsafe_removal_pain_multiplier*I.w_class)//It hurts to rip it out, get surgery you dingus.
```
Comment (Tacolizard) [inline code/modules/mob/living/carbon/human/human.dm]: oh shit i didn't modify it off the latest branch

**Comment by Shadowlight213** [comment]
Nice random revert there buddy.

**Comment by ExcessiveUseOfCobblestone** [comment]
git gud and set them to wanted.

**Comment by Tacolizard** [comment]
OH MY GOD YOU PEOPLE RESPND SO FUCKING FAST

**Comment by Tacolizard** [comment]
let me redo this except on the latest code version

**Pull request closed without merging**